docs/network/interfaces_and_networks.md

```diff
@@ -563,88 +559,18 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
-| Service Mesh out of the box                                    | No<br/>(only ISTIO is supported, adjustmets on both ISTIO and kubevirt had to be done to make it work) | No                                                                        | Yes                                                                                                     |
-| Doesn’t require extra capabilities on the virt-launcher pod    | Yes<br/>(multiple workarounds had to be added to kubevirt to make it work)                             | No<br/>(Multiple workarounds had to be added to kubevirt to make it work) | Yes                                                                                                     |
-| Doesn't require extra network devices on the virt-launcher pod | No<br/>(bridge and tap device are created)                                                             | No<br/>(bridge and tap device are created)                                | Yes                                                                                                     |
-| Supports IPv6                                                  | Yes<br/>(requires manual configuration on the VM)                                                      | No                                                                        | Yes                                                                                                     |
-
-```yaml
-kind: VM
-spec:
-  domain:
-    devices:
-      interfaces:
-        - name: red
-          passt: {} # connect using passt mode
-          ports:
-            - port: 8080 # allow incoming traffic on port 8080 to get into the virtual machine
-  networks:
-    - name: red
-      pod: {}
-```
-
-#### Requirements/Recommendations:
-1. To get better performance the node should be configured with:
-```
-sysctl -w net.core.rmem_max = 33554432
-sysctl -w net.core.wmem_max = 33554432
-```
-2. To run multiple passt VMs with no explicit ports, the node's `fs.file-max` should be increased
-   (for a VM forwards all IPv4 and IPv6 ports, for TCP and UDP, passt needs to create ~2^18 sockets):
-```
-sysctl -w fs.file-max = 9223372036854775807
-```
-
-*NOTE*: To achieve optimal memory consumption with Passt binding, specify ports required for your workload.
-When no ports are explicitly specified, all ports are forwarded, leading to memory overhead of up to 800 Mi.
-
-#### Temporary restrictions: 
-1. `passt` currently only supported as primary network and doesn't allow extra multus networks to be configured on the VM.
-
-passt interfaces are feature gated; to enable the feature, follow
-[these](../cluster_admin/activating_feature_gates.md#how-to-activate-a-feature-gate)
-instructions, in order to activate the `Passt` feature gate (case sensitive).
-
-More information about passt mode can be found in [passt
-Wiki](https://passt.top/passt/about/).
-
-
-
 ### sriov
 
-In `sriov` mode, virtual machines are directly exposed to an SR-IOV PCI
```

Comment:
The SR-IOV change is pretty big.
Please consider performing it in a dedicated commit.

Reply:
done, and passt separated as well